lib/lotus/model.rb

```diff
@@ -25,13 +28,13 @@ class NonPersistedEntityError < ::StandardError
     # @since 0.2.0
     #
     # @see Lotus::Configuration#mapping
-    class InvalidMappingError < ::StandardError
+    class InvalidMappingError < Error
```

Comment:
I added couple of test [there](https://github.com/karimmtarek/model/blob/introduce-lotus-model-error/test/error_test.rb), I just need to know if I'm testing the right thing, can you please look into it and advice?